src/components/paragraphs/stanford-card/card-paragraph.tsx

```diff
@@ -20,18 +22,22 @@ const CardParagraph = ({paragraph, ...props}: Props) => {
 
   const headerTagChoice = (behaviors.su_card_styles?.heading || "h2").split(".", 2);
   const headerTag = headerTagChoice[0]
-  const headerClasses = headerTagChoice[1]?.replace(".", " ").replace("su-font-splash", "text-m2 font-bold")
-
+  const headerClasses = headerTagChoice[1]?.replace(".", " ").replace("su-font-splash", "text-m2 font-normal mb-12") || "font-normal mb-12"
+  const cardVariant = behaviors.su_card_styles?.sum_card_variant;
+  const pillCardBgColor = behaviors.su_card_styles?.sum_card_bg_color_variant;
   const hideHeader = behaviors.su_card_styles?.hide_heading;
 
+  const Element = cardVariant ? PillCard : ImageCard
```

Comment:
Let's check for the value of the variant. We might have more variants in the future.